code/game/objects/items/ego_weapons/he.dm

```diff
@@ -589,6 +589,69 @@
 	if(force < 100)
 		force += 0.25 //It charges pretty slowly, but people walk pretty fast thanks to justice.
 
+/obj/item/ego_weapon/lance/brick_road
+	name = "brick road"
+	desc = "To the wizard we go!"
+	special = "This weapon leaves slowing yellow bricks from charging"
+	icon_state = "brick_road"
+	lefthand_file = 'icons/mob/inhands/96x96_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/96x96_righthand.dmi'
+	damtype = WHITE_DAMAGE
+	armortype = WHITE_DAMAGE
+	force = 50
+	inhand_x_dimension = 96
+	inhand_y_dimension = 96
+	attack_speed = 4//Really really slow.
+	attack_verb_continuous = list("stabs", "impales")
+	attack_verb_simple = list("stab", "impale")
+	hitsound = 'sound/weapons/ego/spear1.ogg'
+	attribute_requirements = list(
+							JUSTICE_ATTRIBUTE = 40
+							)
+	reach = 2
+	force_cap = 100 //Old max damage when it was damage = amount you walked.
+	force_per_tile = 5 //if I can read, this means you need to cross 20 tiles for max damage
+	pierce_force_cost = 20
+	charge_speed_cap = 2//more of a jog then a ram
+	couch_cooldown_time = 5 SECONDS
+
+/obj/item/ego_weapon/lance/brick_road/UserMoved(mob/user)
+	..()
+	if(!raised)
+		playsound(src, 'sound/abnormalities/roadhome/House_MakeRoad.ogg', 100, FALSE, 8)
+		new /obj/effect/golden_road2(get_turf(user))
+
+//Not an actual floor, but an effect you put on top of it.
+/obj/effect/golden_road2
+	name = "Golden Road"
+	icon = 'icons/turf/floors.dmi'
+	icon_state = "gold" //note : find a proper brick road sprite later
+	alpha = 0
+	anchored = TRUE
+	var/slow_down = 0.75//25% slow
+	var/list/faction = list("neutral")
+
+/obj/effect/golden_road2/Initialize()
+	. = ..()
+	QDEL_IN(src, 30 SECONDS)
+	animate(src, alpha = 255,transform= transform, time = 0.5 SECONDS)
+	addtimer(CALLBACK(src, .proc/FadeOut), 29.5 SECONDS)
+	for(var/i = 1 to 30)
+		addtimer(CALLBACK(src, .proc/Slow), i * 1 SECONDS)
+
+/obj/effect/golden_road2/proc/FadeOut()
+	animate(src, alpha = 0, time = 0.5 SECONDS)
+
+/obj/effect/golden_road2/proc/Slow()
+	for(var/turf/T in range(0, src))
+		if(T.z != z)
+			continue
+		for(var/mob/living/L in T)
+			if(faction_check(L.faction, src.faction))
+				continue
+			var/mob/living/simple_animal/hostile/H = L
+			H.TemporarySpeedChange(H.move_to_delay*slow_down , 3 SECONDS)
+
```

Comment:
As discussed it would be better to simply make this apply a unique status effect that increases move_to_delay as opposed to this stacking, as it can result in massive increases, like `3 -> 5.25(+2.25) -> 9.1875(+3.9375) -> 16.078125(+6.890625) -> 13.828125(-2.25) -> 20.26171875 (+8.12109375 - 3.9375)` which is the move_to_delay of a mob with 3 move_to_delay standing in a tile for 6 seconds. This is a 675% increase.